### ISSUE TITLE
[Bug] Logging output wierd for "Processing Failed"

### ISSUE THREAD

**Issue by kossboss** [SiCKRAGETV/sickrage-issues#3179]
# Issue details (low priority)

After the latest updates i have some new logs that look weird.
Since this is just logs its low priority. but I thought it should get fixed for a good total product in the end.

Here is what I see in the interface and in the logs:
https://gist.github.com/kossboss/619996149446978ec78d

Looks like the variable process_fail_message doesnt get filled with the meaningful info intended

# system info

SR Version:	BRANCH: (master) / COMMIT: (638270689679f3958d1b593c329e7c6e5cb7448e) 
SR User:	root
SR Locale:	('en_US', 'UTF-8')
SR Config:	/apps/sickrage/config.ini
SR Database:	/apps/sickrage/sickbeard.db
SR Cache Dir:	/apps/sickrage/cache
SR Log Dir:	/apps/sickrage/Logs
SR Arguments:	['-q', '--daemon', '--nolaunch']
Python Version:	2.7.3 (default, Mar 13 2014, 11:03:55) [GCC 4.7.2]

**Comment by SiCKRAGEB0T** [comment]
## Thanks for the bug/issue report
Please make sure to submit all of the [required information](https://github.com/SiCKRAGETV/sickrage-issues/blob/master/README.md), or we won't be able to help you!
Provide as **much** information as you can such as debug logs, commit hash, and tell us how to replicate the problem.For basic **questions** use the [ [forum] ](https://sickrage.tv/forums/), and for **Feature Requests** [ [Feathub] ](http://feathub.com/SiCKRAGETV/SickRage)
Also make sure the title accurately describes your issue. ("SR not working" etc. is not ok!)

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [x] Post debug logs, either inline (for smaller logs) or using gist)

## Many issues can simply be solved by:
- Making sure you update to the latest version.
- Have you tried turning your device off and on again?
- Using the **search** function to see if this issue has already been reported/solved.
- Do a [GIT reset](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#update-problems-try-this).
- Checking the [ [Wiki] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki) for [ [Guides] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Installation-&-Configuration-Guides), [ [FAQS] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes) [ [Main settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Settings-explained), [ [Show settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Show-settings-explained) [ [Remaining settings] ](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Remaining-settings-explained)
- Shutting down SickRage and removing the `cache/mako` & `cache/sessions` folders.
- If SickRage doesnt find any episodes you probably need to :
 - add a [Scene Exception](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Scene-exceptions-and-numbering)
 - Set the [Default Episode Status](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#newly-aired-shows-are-not-downloading-and-set-to-skippedignored) to `WANTED`
 - Make sure you understand how [quality detection](https://github.com/SiCKRAGETV/sickrage-issues/wiki/Quality-Settings#quality-detectiondetermination) works during searches.
 - Shutdown SickRage and remove the `cache.db` file.
- If you use a QNAP and have SSL errors try this [package](http://apps.qnap.community/11-community/17-qsickrage).
- If you have a `'ascii' codec can't encode character` error, than set your Locale to [UTF-8](https://github.com/SiCKRAGETV/sickrage-issues/wiki/FAQ%27s-and-Fixes#i-have-problems-with-special-characters-%C3%A9-etc-what-can-i-do)

*ps. If you enjoy Sickrage and like the support provided, than please consider making a [donation](https://github.com/SiCKRAGETV/SickRage/wiki/Donations) to keep SickRage alive.*

**Comment by miigotu** [comment]
The log is in reverse.

**Comment by kossboss** [comment]
What do you mean? I just got the first part of that gist from the Log tab (so its reverse because newest is on top there), just select and copy paste.

Also The 2 next parts in the gist, are from the log file (grepping at the "Processing Failed"  and around it 20 lines):

    cat /apps/sickrage/Logs/*log | grep -C20 "Processing Failed:"

**Comment by miigotu** [comment]
The reason is clearly there:
`File exists and new file quality is lower than existing, marking it unsafe to replace`

**Comment by kossboss** [comment]
Lol. Not that. My bad. I should have been more clear. I understand what the log eludes to, its just the way its written I.e. log grammar if you wanna call it thay. Notice how it has colon at the end, but then nothing after. It feels like an incomplete statement. It just looked weird to me and the nitpicker in me wanted it fixed.  Also when you look at the log files after the colon bracket is a newline with just the word "None".. I think that none should be brought into the same line.

I figured if it had something other than "None" it could also have the same issue and be in the log file on a new line but not in the log GUI.

Again super super low priority as its just log output and I understand logs arent supposed to be perfect. But still I figured an easy fix.
